Package.swift

```diff
@@ -75,7 +78,8 @@ let package = Package(
         .linkedFramework("Foundation"),
         .linkedFramework("LocalAuthentication"),
         .linkedFramework("Security"),
-        .linkedFramework("UIKit"),
+        .linkedFramework("AppKit", .when(platforms: [.macOS])),
```

Comment:
Same as above, are we using AppKit on macOS?

Reply:
Yes, we need to use NSWindow.

Reply:
Move the changes to the next PR.